### PR TITLE
Switch on and schedule entity table check job

### PR DESCRIPTION
**Pull request by ericaporter**
**Context**
EntityTableCheck is a job which sends a row_count, a checksum and a timestamp for each entity table sent to Big Query and it’s purpose is to serve as a tool for confirming production database tables match Big Query tables and then help to debug where they don’t match.

**Changes proposed in this pull request**
This PR enables the job and schedules it to run daily at 00:30

**Guidance to review**
As I am not familiar with Claim - please check the scheduling looks correct and advise if there is a better time for the job to run. It might be a good idea to test this on a review app or staging if possible - I can help with this

N.B. 
Wait until Claims is on the latest version of dfe-analytics before merging i.e. 1.11.6 (separate PR for this)

### CODE REVIEW

**Review by slorek — APPROVED** [review]

File: spec/jobs/entity_table_check_job_spec.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+require "rails_helper"
+
+RSpec.describe "EntityTableCheckJob" do
+  describe "#perform" do
+    it "runs the entity table check job" do
+      expect_any_instance_of(DfE::Analytics::EntityTableCheckJob).to receive(:perform)
+
+      DfE::Analytics::EntityTableCheckJob.new.perform
```
Comment (slorek) [inline spec/jobs/entity_table_check_job_spec.rb:8]: This is calling the wrong class. It may be useful to make the naming more distinct.

**Review by slorek — CHANGES_REQUESTED** [review]
Spec needs fixing.

File: app/jobs/dfe_analytics/entity_table_check_job.rb [hunk 2]
```diff
@@ -1,4 +1,4 @@
-class DfE::Analytics::EntityTableCheckJob < CronJob
+class DfeAnalytics::EntityTableCheckJob < CronJob
   self.cron_expression = "30 0 * * *" # Every day at 00:30
```
Comment (ericaporter) [inline app/jobs/dfe_analytics/entity_table_check_job.rb:3]: Is this really correct?

File: spec/jobs/entity_table_check_job_spec.rb (hunk 1 above)
Comment (ericaporter) [reply spec/jobs/entity_table_check_job_spec.rb:8]: Now updated 👍

**Review by slorek — APPROVED** [review]

**Pull request merged**